docs/wallets/decrediton/upgrading-decrediton.md

```diff
@@ -0,0 +1,16 @@
+# Upgrading Decrediton
+
+To upgrade Decrediton, simply download the new version and install it. You do not need to delete your existing version or perform any additional steps. When you open the upgraded version, your existing wallets and accounts will appear. 
+
+## Upgrading steps
+
+1. Download latest version. The latest downloads can be found at [https://decred.org/downloads/](https://decred.org/downloads/)
+or by clicking on the `Update Available` button in the upper right-hand corner of Decrediton.
+    <br>
```

Comment:
Inline html shouldnt be required here, an empty new-line should do the trick.

Reply:
Forgot to remove that. I tried the line and the `<br>` tag actually, and neither introduced the space I was looking for between the image and the text above it. Looks OK anyway,so just removed the `<br>` (which wasn't doing anything).